docs/Deploy/setup-azuredevops.md

```diff
@@ -1,53 +1,136 @@
 # Azure DevOps - Setup Guide
 
-Please complete the following steps at [Configure Azure permissions for ARM tenant deployments](setup-github.md) page before continuing:
+The [AzOps](https://github.com/Azure/AzOps/) (CI/CD) process was initially designed to run on GitHub. However, we recognise that many customers have invested in Azure DevOps and continue to work with it. As AzOps is packaged in a Container, we can leverage it in an Azure DevOps (ADO) pipeline in the same way that we do with a GitHub action.
 
-* Step 3 - Create SPN and Grant Permission
-* Step 5 - Configure your repo to update changes from upstream
+## Supported scenarios
 
-## Implementation notes
+We currently support all AzOps scenarios. However, the Pull Request procedure slightly deviates from the review process in GitHub.
+In its default configuration, Azure DevOps starts the build validation process when a Pull Request is created. To support the same AzOps push scenario, the manual review process (approval) has to be completed before the `AzOps - Push` pipeline will start.
+Azure DevOps `environments` support `Approvals and checks` which we will be leveraged for the Pull Request flow.
 
-The [AzOps](https://github.com/Azure/AzOps/) (CI/CD) process was initially designed to run on GitHub. However, we recognise that many customers have invested in Azure DevOps and continue to work with it. The Azure DevOps (ADO) implementation uses the same Docker image and appropriates it for use in ADO. This is achieved by passing in the correct environment variables for the process to work.
+## Prerequisites
 
-## Supported scenarios
+Please complete the following steps to create a Service Principal with the correct access for use in the Azure deployment pipelines. A PowerShell script is provided below that simplifies this process. The script does require th
+
+You may need to elevate your access to Azure before being able to create a root scoped assignment https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md
+
+1. Create your AzOps service principal
+1. Assign the root scoped Azure RBAC permission
```

Comment:
It's a good point, it could be scoped at another management group. eg. "/providers/Microsoft.Management/managementGroups/ContosoSubOrg"

This probably needs to be added to this page too.  I can make the change and include in the PR
https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md

Reply:
Made the changes to both docs

Reply:
to deploy the reference implementation we need root "/" permission.

Reply:
ok, have clarified this